apps/wallet/src/ui/app/hooks/useGetAllBalance.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type CoinBalance, type SuiAddress } from '@mysten/sui.js';
+import { useQuery, type UseQueryResult } from '@tanstack/react-query';
+
+import { useRpc } from '_hooks';
+type GetCoinBalanceProps = {
+    address: SuiAddress | null;
+};
+export function useGetAllBalance({
+    address,
+}: GetCoinBalanceProps): UseQueryResult<CoinBalance[], unknown> {
+    const rpc = useRpc();
+    const response = useQuery(
+        ['get-all-balance', address],
+        () => rpc.getAllBalances(address!),
+        // refetchInterval is set to 2 seconds
```

Comment:
Looks like it's actually 20 seconds in code? I think 2 seconds is a bit too often potentially.

Reply:
I will change it